### PR TITLE
test: issue with duplicated prefix for checkpoints on cloud

### PR DESCRIPTION
**Pull request by aryarm**
Implements a test to reproduce issue #574: checkpoints in cloud execution mode.

### QC

* [x] The PR contains a test case for the changes or the changes are already covered by an existing test case.
* [ ] The documentation (`docs/`) is updated to reflect the changes or this is not necessary (e.g. if the change does neither modify the language nor the behavior or functionalities of Snakemake).

### CODE REVIEW

**Comment by aryarm** [comment]
@johanneskoester , I'm not really sure why the new test passed 😕 

It didn't pass when I tried it locally...

File: tests/test_google_lifesciences.py [hunk 1]
```diff
@@ -90,3 +90,22 @@ def test_touch_remote_prefix():
...
+    create_google_storage(bucket_name)
+    storage_prefix = "test_cloud_checkpoints_issue574"
+    workdir = dpath("test_cloud_checkpoints_issue574")
+    try:
+        run(
+            workdir,
+            use_conda=True,
+            default_remote_prefix="%s/%s" % (bucket_name, storage_prefix),
+            google_lifesciences=True,
+            google_lifesciences_cache=True,
```
Comment (aryarm) [inline tests/test_google_lifesciences.py:108]: ah, I think this should be changed based on the other functions in this file? That might be a mistake on my part.
```suggestion
            google_lifesciences_cache=False,
```

Comment (johanneskoester) [reply tests/test_google_lifesciences.py:108]: Yes, this should be False indeed.

**Comment by aryarm** [comment]
hmm the checks seem to be passing, still :(
But on second thought, it doesn't look like the GLS tests are actually getting executed in the first place.

Maybe it's because the PR needs the `update-container-image` tag as described in https://github.com/snakemake/snakemake/pull/691?

**Comment by johanneskoester** [comment]
The problem is that you are on a fork. forks don't have access to all the necessary secrets. Let me do the following: you become part of the Snakemake organization, then you can re-issue your PR from within the original repo, not as a fork. Would that be ok for you?

**Comment by aryarm** [comment]
Sure, that sounds good to me. As a long time user of Snakemake, I really value having the opportunity to help contribute to it!

**Comment by johanneskoester** [comment]
Awesome, you have an invitation now.

**Comment by aryarm** [comment]
closing in favor of #1294

**Pull request closed without merging**